esinstall/package.json

```diff
@@ -56,5 +56,9 @@
     "rollup-plugin-node-polyfills": "^0.2.1",
     "validate-npm-package-name": "^3.0.0",
     "vm2": "^3.9.2"
+  },
+  "devDependencies": {
+    "@types/es-module-lexer": "^0.3.0",
+    "@types/node": "^14.14.14"
```

Comment:
Can you move this to the top-level package.json? All packages should share the same Node types

Reply:
OK，I moved it